solvation_analysis/analysis.py

```diff
@@ -0,0 +1,127 @@
+# from abc import ABC
+from abc import ABC
```

Comment:
Are you using an abstract base class anywhere?

Reply:
Nope, for some reason that got put in by the IDE, I'll take it out.